refactor(deps): migrate to php-cs-fixer v3.0

PedroTroller opened the pull request.

shouze commented:
@PedroTroller cs fixer 3.0 has been released, how can I help making it compatible?

PedroTroller commented:
> @PedroTroller cs fixer 3.0 has been released, how can I help making it compatible?

@shouze by sending me this comment to slap the back of my head :wink:  , thank you, I have rebased my PR and corrected the last few points.

PedroTroller commented:
:tada: This PR is included in version 2.25.0 :tada:

The release is available on [GitHub release](https://github.com/PedroTroller/PhpCSFixer-Custom-Fixers/releases/tag/v2.25.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
